test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -57,6 +57,49 @@
         x "x-"]
     (str r w x)))
 
+(def attribute-file-extensions
+  "These are the extensions defined in csr_attributes.yaml."
+  [{:oid      "1.3.6.1.4.1.34380.1.1.1"
+    :critical false
+    :value    "ED803750-E3C7-44F5-BB08-41A04433FE2E"}
+   {:oid      "1.3.6.1.4.1.34380.1.1.1.4"
+    :critical false
+    :value    "I am undefined but still work"}
+   {:oid      "1.3.6.1.4.1.34380.1.1.2"
+    :critical false
+    :value    "thisisanid"}
+   {:oid      "1.3.6.1.4.1.34380.1.1.3"
+    :critical false
+    :value    "my_ami_image"}
+   {:oid      "1.3.6.1.4.1.34380.1.1.4"
+    :critical false
+    :value    "342thbjkt82094y0uthhor289jnqthpc2290"}
+   {:oid      "1.3.6.1.4.1.34380.1.1.5" :critical false
```

Comment:
couple formatting nitpicks:
* newline before `:critical false`
* our formatting standard is to not align map values: https://github.com/puppetlabs/pl-clojure-style#aligning-values-in-maps-and-lets (although, yeah, I realize there's some older code in this repo written in that style)